### PR TITLE
Add test for Action struct

### PR DESCRIPTION
**Pull request by dairyo**
In #376, We rename `AliasAction.action` to `AliasAction.actions`. We also change type of its property to `Action` which is newly introduced struct.

In this PR

* Add test for `Action`
* Fix test concerned with `AliasAction`.

Related PR: #296

### CODE REVIEW

**Review by naoki-akimoto — APPROVED** [review]
LGTM!

File: ThingIFSDK/ThingIFSDK/AliasAction.swift [hunk 1]
```diff
@@ -29,4 +29,15 @@ public struct AliasAction {
         self.alias = alias
         self.actions = actions
     }
+
+    /** Initializer of AliasAction instance.
+
+     - Parameter alias: Name of an alias.
+     - Parameter actions: Actions of this alias.
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/AliasAction.swift]: `Actions of this alias.` > `Array of Action instances.`

**Comment by dairyo** [comment]
@yongpingchen Fixed according to your comment. Please confirm.

**Review by yongpingchen — APPROVED** [review]
LGTM

**Pull request merged**